R/tm_t_exposure.R

```diff
@@ -428,6 +440,7 @@ ui_t_exposure <- function(id, ...) {
         data_extract_spec = a$row_by_var,
         is_single_dataset = is_single_dataset_value
       ),
+      ui_decorate_teal_data(ns("decorator"), decorators = select_decorators(a$decorators, "table")),
       checkboxInput(ns("add_total_row"), "Add Total row", value = a$add_total_row),
       checkboxInput(ns("add_total"), "Add All Patients column", value = a$add_total),
```

Comment:
Maybe UI is better moved after the modules' input, or even after the collapsed "Additional Variables Info"

WDYT?

```suggestion
      checkboxInput(ns("add_total_row"), "Add Total row", value = a$add_total_row),
      checkboxInput(ns("add_total"), "Add All Patients column", value = a$add_total),
      ui_decorate_teal_data(ns("decorator"), decorators = select_decorators(a$decorators, "table")),
```

Reply:
I don't think you need to group them, but I leave that to you

Reply:
I meant below it :facepalm: your comment still applies as that is just "info" and a decorator might take precedent.